users/ignoring.rst

```diff
@@ -91,8 +91,13 @@ The ``.stignore`` file contains a list of file or path patterns. The
    :ref:`watch <scanning>` the entire directory tree regardless of other
    ignore patterns.
 
-Example
--------
+   Top-level Include patterns (beginning with ``/`` and matching at the root of
+   the folder only) are treated as special cases and will not force Syncthing to
```

Comment:
I'd drop the the text in parentheses, the examples in the end should make it clear already.